### ISSUE TITLE
Condarc ignored if not ending in condarc when platform is specified

### ISSUE THREAD

**Issue by gabrielsimoes** [mamba-org/mamba#2332]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Search tried in issue tracker

...

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not reproducible with Conda

### Describe your issue

See info logs and condarc contents for details.
```
CONDARC=condarc micromamba info
```
works as expected, but:
```
CONDARC=file.txt micromamba info
```
has empty channels, even though file.txt is exactly the same file as condarc. This also happens when using `--rc-file` instead of the `CONDARC` environment variable.

### mamba info / micromamba info

```shell
$ CONDARC=file.txt micromamba info

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/

            environment : None (not found)
           env location : -
      user config files : /Users/gsilvasimoes/.mambarc
 populated config files :
       libmamba version : 1.3.1
     micromamba version : 1.3.1
           curl version : libcurl/7.86.0 (SecureTransport) LibreSSL/3.3.6 zlib/1.2.11 nghttp2/1.47.0
     libarchive version : libarchive 3.5.3 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.8
       virtual packages : __unix=0=0
                          __osx=13.2.1=0
                          __archspec=1=arm64
               channels :
       base environment : /Volumes/git/mamba
               platform : osx-arm64

$ CONDARC=condarc micromamba info

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/


            environment : None (not found)
           env location : -
      user config files : /Users/gsilvasimoes/.mambarc
 populated config files : condarc
       libmamba version : 1.3.1
     micromamba version : 1.3.1
           curl version : libcurl/7.86.0 (SecureTransport) LibreSSL/3.3.6 zlib/1.2.11 nghttp2/1.47.0
     libarchive version : libarchive 3.5.3 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.8
       virtual packages : __unix=0=0
                          __osx=13.2.1=0
                          __archspec=1=arm64
               channels : https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/conda-forge/noarch
       base environment : /Volumes/git/mamba
               platform : osx-arm64
```

### ~/.condarc

```yaml
default_channels: []
channels:
    - conda-forge
```

**Comment by gabrielsimoes** [comment]
Closing as this behavior actually comes from conda itself.

**Comment by jonashaag** [comment]
Hi @gabrielsimoes can you please give feedback to the issue template? Always looking to improve it. 

1. You said that this wasn’t reproducible with Conda although it is. Did you try reproducing with Conda before opening the issue?
2. I reported this issue a while ago so it is a duplicate. Did you do a search before reporting this issue?
3. 

Thank you!

**Comment by gabrielsimoes** [comment]
Hey @jonashaag, both overlooks on my end, I think:
1. Conda has a slightly more general check `"condarc" in filename`, so initially it was working when I used a file like `something-condarc`. When I cooked an example for the bug report I started using completely unrelated names like "file.txt", but I forgot to check with conda again.
2. I did a search before, but did not find your issue. I guess it was not clear to me then that the issue was related to condarc.
The template is comprehensive IMO/I don't have any feedback.
